mesheryctl/internal/cli/root/system/model.go

```diff
@@ -446,6 +446,12 @@ mesheryctl exp model view [model-name]
 		}
 		return nil
 	},
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return cmd.Help()
```

Comment:
```suggestion
			return cmd.Help()
```
You can return an error message as @RipulHandoo said. Eg: `This command takes at least one arg.` and then you could use `cmd.Help()` to display the help message